src/main/java/hudson/remoting/Request.java

```diff
@@ -383,7 +383,11 @@ public void run() {
                 } catch (IOException e) {
                     // communication error.
                     // this means the caller will block forever
-                    logger.log(Level.WARNING, "Failed to send back a reply to the request " + this, e);
+                    if (e instanceof ChannelClosedException) {
+                        logger.log(Level.INFO, "Failed to send back a reply to the request {0}: {1}", new Object[] {this, e});
```

Comment:
I would retain the original behavior on the FINE level so that we can locate what caused Closed in some cases.
Or maybe you could retrieve the `cause` message via new API method or `getMessage()` update in `ChannelStateException `

Reply:
> retrieve the `cause` message

Should not be necessary. `Throwable.toString` by default prints the `toString` of a cause.